[2025-03-17T19:04:38.576+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T19:04:38.648+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_tables.events_project_c_db manual__2025-03-17T19:04:34.626694+00:00 [queued]>
[2025-03-17T19:04:38.708+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_tables.events_project_c_db manual__2025-03-17T19:04:34.626694+00:00 [queued]>
[2025-03-17T19:04:38.717+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-17T19:04:38.889+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): load_tables.events_project_c_db> on 2025-03-17 19:04:34.626694+00:00
[2025-03-17T19:04:38.937+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2533) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-17T19:04:38.940+0000] {standard_task_runner.py:72} INFO - Started process 2549 to run task
[2025-03-17T19:04:38.950+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_tables.events_project_c_db', 'manual__2025-03-17T19:04:34.626694+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp3d3vqfnn']
[2025-03-17T19:04:38.959+0000] {standard_task_runner.py:105} INFO - Job 182: Subtask load_tables.events_project_c_db
[2025-03-17T19:04:39.837+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag.load_tables.events_project_c_db manual__2025-03-17T19:04:34.626694+00:00 [running]> on host c032966e105e
[2025-03-17T19:04:40.517+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_tables.events_project_c_db' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T19:04:34.626694+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-17T19:04:34.626694+00:00'
[2025-03-17T19:04:40.537+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-17T19:04:40.729+0000] {sql.py:278} INFO - Executing: select load_tables('events_project_c_db', 'events');
[2025-03-17T19:04:40.829+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T19:04:40.976+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T19:04:41.078+0000] {sql.py:553} INFO - Running statement: select load_tables('events_project_c_db', 'events');, parameters: None
[2025-03-17T19:04:42.669+0000] {sql.py:562} INFO - Rows affected: 1
[2025-03-17T19:04:42.916+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-17T19:04:42.947+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_tables.events_project_c_db, run_id=manual__2025-03-17T19:04:34.626694+00:00, execution_date=20250317T190434, start_date=20250317T190438, end_date=20250317T190442
[2025-03-17T19:04:43.087+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T19:04:43.220+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-17T19:04:43.358+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
